SEUtils/SEUtils/Mat22.h

```diff
@@ -0,0 +1,106 @@
+/*
+ * Mat22.h
+ *
+ *  Created on: Jul 25, 2019
+ *      Author: mschefer
+ */
+
+#ifndef _SEUTILS_MAT22_H_
+#define _SEUTILS_MAT22_H_
+
+#include <tuple>
+#include <assert.h>
+
+namespace SExtractor {
+
+class Mat22;
```

Comment:
I wonder if it would make sense to move this class eventually to Alexandria :thinking: